src/hooks/usePokemon/usePokemon.js

```diff
@@ -0,0 +1,17 @@
+import { useState, useGlobalState, useEffect } from "tram-one"
+import Pokedex from 'pokedex-promise-v2'
+const PokeAPI = new Pokedex()
+
+export default () => {
+  const [pokemonName, setPokemonName] = useState('')
```

Comment:
this may eventually need to be useGlobalState? I actually don't know why this works as is... because multiple calls will have this as `''`, but maybe that won't matter because you won't ever really reset `pokemon`